src/components/playerInput/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useState } from 'react';
+
+import './style/player.css';
+
+export default function PlayerInput({ children }) {
+  return <form className="inputForm">{children}</form>;
```

Comment:
should the form really wrap only the player input!?
Later you want also to have inputs for setup game parameters (points per set and other). How you want to handle this?

Reply:
Should I rename it, "gameInput" because is more about the game than the player itseld, right?

Reply:
that's a good solution 👍